.github/workflows/fedora-copr-build.yml

```diff
@@ -128,7 +128,7 @@ jobs:
             --description  "$(cat project-description.md)" \
             --unlisted-on-hp on \
             --enable-net on \
-            --runtime-repo-dependency "https://download.copr.fedorainfracloud.org/results/%40fedora-llvm-team/llvm-compat-packages/\$distname-\$releasever-\$basearch" \
+            --runtime-repo-dependency "https://download.copr.fedorainfracloud.org/results/%40fedora-llvm-team/llvm-compat-packages/\$chroot" \
```

Comment:
Which project processes `$chroot`?
I'm aware the previous variables were part of the definition of YUM repos, but I couldn't find references to `$chroot`.